src/sentry/digests/notifications.py

```diff
@@ -223,15 +225,14 @@ def get_rules_from_workflows(project: Project, workflow_ids: set[int]) -> dict[i
 
             assert rule.project_id == project.id, "Rule must belong to Project"
 
-            if should_fire_workflow_actions(project.organization):
-                rule.data["actions"][0]["legacy_rule_id"] = rule.id
+            # if should_fire_workflow_actions(project.organization):
+            rule.data["actions"][0]["legacy_rule_id"] = rule.id
```

Comment:
It seems like `rule.data["actions"]` isn't guaranteed to exist (tests are failing), so we should check for it. I'm also curious why we only set legacy rule id on the first block (I think)?